### PR TITLE
App/pinchflat

### PR DESCRIPTION
**Pull request by falkheiland**
https://github.com/kieraneglin/pinchflat

Pinchflat

Your next YouTube media manager

Pinchflat is a self-hosted app for downloading YouTube content built using yt-dlp. It's designed to be lightweight, self-contained, and easy to use. You set up rules for how to download content from YouTube channels or playlists and it'll do the rest, checking periodically for new content. It's perfect for people who want to download content for use in with a media center app (Plex, Jellyfin, Kodi) or for those who want to archive media!

## Summary by CodeRabbit

- **New Features**
	- Updated `config.json` with detailed configuration settings for Pinchflat application.
	- Introduced Docker Compose configuration in `docker-compose.yml` for simplified deployment.
	- Added `description.md` providing an overview and setup instructions for Pinchflat app.

- **Documentation**
	- Added detailed app description and setup guidelines in `description.md`.

### CODE REVIEW

File: apps/pinchflat/metadata/description.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Pinchflat
```
[the remark this answers is not in the document]
Comment (falkheiland) [reply apps/pinchflat/metadata/description.md:1]: copy paste from project side, won't change

File: apps/pinchflat/metadata/description.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Your next YouTube media manager
+
+Pinchflat is a self-hosted app for downloading YouTube content built using yt-dlp. It's designed to be lightweight, self-contained, and easy to use. You set up rules for how to download content from YouTube channels or playlists and it'll do the rest, checking periodically for new content. It's perfect for people who want to download content for use in with a media center app (Plex, Jellyfin, Kodi) or for those who want to archive media!
```
[the remark this answers is not in the document]
Comment (falkheiland) [reply apps/pinchflat/metadata/description.md:5]: copy paste from project side, won't change

**Comment by MontejoJorge** [comment]
The docker image suports arm64 too, could you add it on the config.json?

**Comment by falkheiland** [comment]
> The docker image suports arm64 too, could you add it on the config.json?

done

**Comment by meienberger** [comment]
Thank you @falkheiland, I had to make a few [adjustments](https://github.com/runtipi/runtipi-appstore/commit/3e0ae4eb3eef90df4bc4b0aab02dc81ca94a3204) to your config

**Pull request merged**